src/test/java/Homework16.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Navigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "http://bbb.testpro.io/";
+        driver.get(url);
```

Comment:
Everything is good. But your pull request contains the file 'Base Page' and it @BeforeMethod opens the Url/
I can suggest that your pull request contains some unnecessary code ( For HW 16 )